pennylane/devices/tests/test_templates.py

```diff
@@ -225,6 +225,15 @@ def circuit():
             [math.fidelity_statevector(circuit(), exp_state)], [1.0], atol=tol(dev.shots)
         )
 
+    @pytest.fixture(scope="function", autouse=True)
+    def capture_warnings(self, recwarn):
+        """Capture warnings."""
+        yield
+        if len(recwarn) > 0:
+            for w in recwarn:
+                assert isinstance(w.message, qml.PennyLaneDeprecationWarning)
+                assert "BasisStatePreparation is deprecated" in str(w.message)
+
     def test_BasisStatePreparation(self, device, tol):
```

Comment:
We can just update this test to use `qml.BasisState` instead of `qml.BasisStatePreparation`.

Reply:
Solved with `autouse = False` 😄